### DISCUSSION TITLE
merging of work account contribution into personal account

### DISCUSSION THREAD

**Discussion by ransayada** [community/community#55718] [New to GitHub] [answered]
I used this topic [https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/merging-multiple-personal-accounts](url) in order to merge my Work Account with my Personal Account, 
however, all my contributions are gone now, and I already deleted my work account for that process. 
Is there any chance I can retrieve my contributions? or what did I do wrong that my contributions do not migrate properly?
thanks

**Comment by airtower-luna** [comment]
Did you add the email address(es) of the deleted account to the one you want to keep? Then your contributions should show up.

Edit: Also keep in mind that this will only get the *commits* you made to carry over. Other things (PRs, issues, etc.) won't.

**Reply by ransayada** [reply]
thanks for the fast replay, 
I did add all the email addresses I used on my work account (the account I deleted) into the personal account (the account I want to stay with) like it written in the docs, however, the contributions stay the same (only the contributions from the personal account without the new ones from the work account), I know it only the commits but the commits are contributions as well.
any other suggestions?

**Reply by airtower-luna** [reply]
When you check the repositories you committed using those old addresses, are the commits shown as yours, as in your current account? If yes, it might simply be a caching issue. If not, it'll need a closer look at those commits, in particular: What's the actual author email in them?

**Reply by ransayada** [reply]
the commits are shown as mine, if it is a caching issue there is something I can do? 
this is the email I used for those commits: ran@bit-tax.com

**Reply by airtower-luna** [reply, marked as answer]
Not really. I'd wait 2 days (from when you added the address), and if the issue still exists [report it to GitHub support](https://support.github.com/request), so they can take a look and maybe flush/rebuild the cache.

**Reply by ransayada** [reply]
well, I sent a ticket to the support team, hope that they can help.
thank you for your kind replays :)
